### PR TITLE
CB-10178 Fix a number of documentation issues

### PR DESCRIPTION
**Pull request by nikhilkh**
Here are the list of fixes
- Add more examples for icons for Windows and link to platform SDK docs.
- Ensure all references to CLI guide - now reference to notion of "creating your first app"
- Update all links for plugins to point to internal docs as opposed to npm/GitHub.

### CODE REVIEW

**Comment by nikhilkh** [comment]
@rakatyal

File: www/docs/en/dev/guide/hybrid/plugins/index.md [hunk 1]
```diff
@@ -93,30 +95,28 @@ the platform-specific `config.xml` file to make the platform aware of
...
+```
 
 You need an valid app source directory, such as the top-level `www`
 directory included in a default CLI-generated project as described in
-[The Command-Line Interface](../../cli/index.html).  Make sure the app's `index.html` home
-page reference the name of the plugin's JavaScript interface, as if it
-were in the same source directory:
-
-        <script src="myplugin.js"></script>
+[Create your first app guide](../../cli/index.html).
```
Comment (rakatyal) [inline www/docs/en/dev/guide/hybrid/plugins/index.md]: Remove 'guide' as part of the link.

File: www/docs/en/dev/guide/overview/index.md [hunk 2]
```diff
@@ -108,17 +106,15 @@ task, they each offer advantages:
...
   for many platforms at once, abstracting away much of the functionality of 
   lower-level shell scripts. The CLI copies a common set of web assets into
   subdirectories for each mobile platform, makes any necessary
   configuration changes for each, runs build scripts to generate
   application binaries. The CLI also provides a common interface to
-  apply plugins to your app. For more details on the CLI, see The
-  Command-Line Interface. Unless you have a need for the platform-centered
-  workflow, the cross-platform workflow is recommended.
+  apply plugins to your app. To get started follow the steps in the 
+  [Create your first app tutorial](../cli/index.html). Unless you have a need for the platform-centered workflow, the cross-platform workflow is recommended.
```
Comment (rakatyal) [inline www/docs/en/dev/guide/overview/index.md]: Same. Remove 'tutorial' as part of the link.

File: www/docs/en/dev/guide/overview/index.md [hunk 3]
```diff
@@ -129,27 +125,20 @@ task, they each offer advantages:
...
   apps, it is generally more difficult because the lack of a
   higher-level tool means separate build cycles and plugin
-  modifications for each platform. Still, this workflow allows you
-  greater access to development options provided by each SDK, and is
-  essential for complex hybrid apps.
+  modifications for each platform. 
 
 When first starting out, it may be easiest to use the cross-platform
-workflow to create an app, as described in [The Command-Line Interface](../cli/index.html).
+workflow to create an app, as described in [Create your first app tutorial](../cli/index.html).
```
Comment (rakatyal) [inline www/docs/en/dev/guide/overview/index.md]: Here too.

**Comment by rakatyal** [comment]
LGTM!

File: www/docs/en/dev/config_ref/images.md [hunk 4]
```diff
@@ -42,79 +45,107 @@ density       | *Optional* <br/> Specified icon density (Android Specific)
...
+        <icon src="res/android/mdpi.png" density="mdpi" />
+        <icon src="res/android/hdpi.png" density="hdpi" />
+        <icon src="res/android/xhdpi.png" density="xhdpi" />
+        <icon src="res/android/xxhdpi.png" density="xxhdpi" />
+        <icon src="res/android/xxxhdpi.png" density="xxxhdpi" />
+    </platform>
+```
+###See Also
+- [Android icon guide](https://www.google.com/design/spec/style/icons.html)
+- [Android - Supporting multiple screens](http://developer.android.com/guide/practices/screens_support.html)
```
Comment (dblotsky) [inline www/docs/en/dev/config_ref/images.md]: Please use reference-style links.

**Comment by dblotsky** [comment]
LGTM.

File: www/docs/en/dev/config_ref/images.md (hunk 4 above)
Comment (nikhilkh) [inline www/docs/en/dev/config_ref/images.md]: Can you comment on why it is better in this case?

Comment (dblotsky) [inline www/docs/en/dev/config_ref/images.md]: It's easier to see them all in the same place at the bottom, but it doesn't make much of a difference.

**Pull request closed without merging**